PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/README.md

```diff
@@ -113,7 +113,12 @@ integration, telemetry connectivity, and Azure Toolkit integration.
 7. #### Use SyncPoller instead of PollerFlux#getSyncPoller()
 
 - **Anti-pattern**: Using `getSyncPoller()` on a `PollerFlux` instance is an anti-pattern.
-- **Issue**: The main issue with using `getSyncPoller()` is that it introduces additional complexity by converting an asynchronous polling mechanism to a synchronous one, which should be avoided.
+- **Issue**: The main issue with using `getSyncPoller()` is that it introduces additional complexity by converting an
+  asynchronous polling mechanism to a synchronous one, which should be avoided.
 - **Severity: WARNING**
-- **Recommendation**: Instead of using `getSyncPoller()`, it's recommended to use the `SyncPoller` directly to handle synchronous polling tasks. `SyncPoller` provides a synchronous way to interact with the poller and is the preferred method for synchronous operations.
-  Please refer to the [Azure SDK for Java documentation](https://learn.microsoft.com/java/api/com.azure.core.util.polling.syncpoller?view=azure-java-stable) for additional information.
\ No newline at end of file
+- **Recommendation**: Instead of using `getSyncPoller()`, it's recommended to use the `SyncPoller` directly to handle
```

Comment:
why this diff?

Reply:
what do you mean by diff? the line breaks? I used Ctrl+Alt+L (shortcut to format document) when I finished

Reply:
This isnt related to the PR I would remove the change diff.